Adjust the text used in the install help message and man page to mention
the OpenIndiana specific packaging tool.

This patch has not been passed upstream yet. It needs to be reworked into
a format that would be acceptable.

--- coreutils-9.5/src/install.c.orig
+++ coreutils-9.5/src/install.c
@@ -585,8 +585,8 @@
 \n\
 This install program copies files (often just compiled) into destination\n\
 locations you choose.  If you want to download and install a ready-to-use\n\
-package on a GNU/Linux system, you should instead be using a package manager\n\
-like yum(1) or apt-get(1).\n\
+package on an OpenIndiana system, you should instead be using a package manager\n\
+like pkg(1).\n\
 \n\
 In the first three forms, copy SOURCE to DEST or multiple SOURCE(s) to\n\
 the existing DIRECTORY, while setting permission modes and owner/group.\n\
